src/serialize.jl

```diff
@@ -0,0 +1,27 @@
+using ..ChemistryFeaturization.AbstractType: AbstractAtoms, AbstractFeaturization
+
+"""
+    SerializableEncodedFeature
```

Comment:
Personally, I prefer something like `FeaturizedAtoms` as the name. It's a bit more general and descriptive of what it actually is rather than just one intended purpose of it, and there's nothing about the `Atoms` objects that makes them _not_ serializable...

Reply:
Also, it stores multiple encoded features, not just one...

Reply:
In that case, `FeaturizedAtoms` it is! I wasn't really attached to `SerializableEncodedFeature` anyways #ntih

Reply:
Resolved in 18893cf9dad7d546f3eff93a005abe212957440d.